deploy.js

```diff
@@ -46,12 +46,13 @@ async function registerCommands() {
     },
     {
       name: "playlist",
-      description: "Plays all songs from a youtube playlist.",
+      description:
+        "Plays all songs from a youtube playlist or spotify album or playlist.",
       options: [
         {
           name: "playlisturl",
           type: "STRING",
-          description: "Youtube URL to playlist",
+          description: "Link to youtube playlist or spotify album or playlist.",
```

Comment:
Is there a difference between a Spotify album or playlist?

Reply:
No but it's just to clarify that you can do either with that command.